.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+# Changes
+
+<!-- 
+Describe your changes here- ideally you can get that description straight from your descriptive commit message(s)! 
+
+In addition, categorize the changes you're making using the "/kind" Prow command, example:
+
+/kind <kind>
+
+Supported kinds are: bug, test, documentation, enhancement
+-->
+
+# Submitter Checklist
+
+As the author of this PR, please check off the items in this checklist:
+
+- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
+  included if any changes are user facing
+- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
+  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
+- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
```

Comment:
If we're going to copy-paste the commit message standard from tekton, we might as well link to our own copy directly: 
```suggestion
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)
```

Reply:
done.